docs/scenarios_authoring/Fragments.md

```diff
@@ -23,3 +23,4 @@ Fragment can define zero, one or more outputs. Each of them has a name, main sce
 - They cannot be nested (i.e. fragment cannot invoke other fragment).
 - They cannot pass output variables to the main scenario. This may change in the future.
 - When inputs/outputs of fragment change, scenarios using it have to be corrected manually.
+- If fragment use some component which clears context, context will be also cleared in the main scenario.
```

Comment:
@ZbyszekMM do you get what context is ?

Reply:
We should either describe what context is (here above or in Intro) or refer to I rather simply as `variables`

Reply:
@dswiecki I agree, `context` term could be obscure to readers of this doc.
See new changes if they are sufficient